testsuite/integration/basic/pom.xml

```diff
@@ -3779,8 +3772,9 @@
                                         <exclude>org/jboss/as/test/integration/jsf/version/JSFDeploymentProcessorTestCase.java</exclude>
                                         <exclude>org/jboss/as/test/integration/management/deploy/runtime/JaxrsRuntimeNameTestCase.java</exclude><!-- test expects javax strings in mgmt api responses -->
                                         <exclude>org/jboss/as/test/integration/weld/ejb/SessionObjectReferenceTestCase.java</exclude>
-                                        <exclude>org/wildfly/test/integration/microprofile/jwt/ejb/JWTEJBTestCase.java</exclude><!-- No JWT. TODO why would JWT work in the normal profile? -->
                                         <exclude>org/wildfly/test/integration/web/annotationsmodule/EarModuleDeploymentTestCase.java</exclude>
+                                        <exclude>org/wildfly/test/integration/observability/opentelemetry/RestClientTestCase</exclude>
+                                        <exclude>org/wildfly/test/integration/observability/opentelemetry/BasicOpenTelemetryTestCase</exclude>
```

Comment:
Please remove these; https://github.com/wildfly/wildfly/pull/14674 should fix this problem.

Reply:
Done @bstansberry